.github/workflows/build_doc.yml

```
@@ -109,7 +109,7 @@ jobs:
             mv tmp.html index.html
             git add ${PR_NUMBER}/$ROUND index.html && git commit -q --amend -m "base commit" && git push -q -f -u origin master
           else
-            echo "::set-output name=DoxygenError::This round already exists. Overwrite it with /force-build"
+            echo "::set-output name=DoxygenError::This round already exists. Overwrite it with /force-build."
```

We can even display the full command, with the `:round` part:
```suggestion
            echo "::set-output name=DoxygenError::This round already exists. Overwrite it with /force-build:${ROUND}."
```

---

I think it'd be nice if these "build_doc" commands actually contained "doc" in the name too.

---

I agree. It should be build-doc:v1 and force-build-doc:v1.

I cannot add the `/` in my comment, otherwise our stupid action bot would trigger the action. I would be safer if the action bot only reacted when the command is alone in the message, or at least alone in the first line of the message.